src/components/CompareResults/RunValues.tsx

```diff
@@ -41,11 +42,12 @@ function RunValues(props: RunValuesProps) {
     <div className={styles.container}>
       {application ? (
         <div>
-          <b>{name}:</b> {median} {measurementUnit} ({application})
+          <b>{name}:</b> {formatNumber(median)} {measurementUnit} ({application}
+          )
         </div>
       ) : (
         <div>
-          <b>{name}:</b> {median} {measurementUnit}
+          <b>{name}:</b> {formatNumber(median)} {measurementUnit}
```

Comment:
it's a bug that we were using median here, can you please move to the `avg` value instead?